source/js/scripts/app.js

```diff
@@ -36,6 +36,10 @@ import * as backend from '../backend.js';
  * }
  */
 
+// Icon assets
+const pomoIcon = require('url:../../img/green-tomato.png');
+const breakIcon = require('url:../../img/red-tomato.png');
```

Comment:
Can we not use module imports here? I.e., `import pomoIcon from '../../img/green-tomato.png'`?

Reply:
Probably, I'll look into it